### PR TITLE
Bug fixes + Improvements

### PR DESCRIPTION
**Pull request by todi1856**
Bug fixes:
https://fogbugz.unity3d.com/f/cases/1172309/ Provide name for  Stacktrace Utility window
https://fogbugz.unity3d.com/f/cases/1172331/ Right clicking on log will select it, like it's done in other Unity windows, for ex., Project View
https://fogbugz.unity3d.com/f/cases/1172335/ Ctrl + C will copy logs as expected
https://fogbugz.unity3d.com/f/cases/resolve/1171501/ Column separators should draw correctly now

Improvements:
Added Android Logcat Settings

AndroidLogcatSettings

### CODE REVIEW

File: com.unity.mobile.android-logcat/Editor/AndroidLogcatRuntime.cs [hunk 1]
```diff
@@ -28,6 +30,7 @@ internal interface IAndroidLogcatRuntime
     internal class AndroidLogcatRuntime : IAndroidLogcatRuntime
     {
         private AndroidLogcatDispatcher m_Dispatcher;
+        private AndroidLogcatSettings m_Settings;
```
Comment (vincent-unity) [inline com.unity.mobile.android-logcat/Editor/AndroidLogcatRuntime.cs]: Just thinking if we should have two settings: one for dark skin, and one for light skin? Or we store two sets in one AndroidLogcatSettings? As with different skin, users might want to choose different colors.

When dealing with device simulator UI, I met the case that I have to write different uss styles for both dark and light skins, then load the right one by checking `EditorGUIUtility.isProSkin`.

**Comment by vincent-unity** [comment]
Nice job! The AndroidLogcatSettings looks super cool!

File: com.unity.mobile.android-logcat/Editor/AndroidLogcatRuntime.cs (hunk 1 above)
Comment (todi1856) [reply com.unity.mobile.android-logcat/Editor/AndroidLogcatRuntime.cs]: Other settings with colors don't do that. Let' see any of users will complain about this, before proceeding. It's not a very common scenario.

I agree though that window styles, having two sets of styles, makes sense

Comment (vincent-unity) [reply com.unity.mobile.android-logcat/Editor/AndroidLogcatRuntime.cs]: 👌

**Comment by WashingMachinae** [comment]
Bug fixes:
https://fogbugz.unity3d.com/f/cases/1172309/ Provide name for Stacktrace Utility window

**Good, window name looks proper now.**

https://fogbugz.unity3d.com/f/cases/1172331/ Right clicking on log will select it, like it's done in other Unity windows, for ex., Project View
**Right clicking row selects it now, but some other issue has appeared**
#42 

https://fogbugz.unity3d.com/f/cases/1172335/ Ctrl + C will copy logs as expected
**Good** 

https://fogbugz.unity3d.com/f/cases/resolve/1171501/ Column separators should draw correctly now
**Good**

Improvements:
Added Android Logcat Settings
**Issues found with the Settings:**
#41 , #44

**Pull request merged**